service.go

```diff
@@ -21,8 +21,23 @@ import (
 
 var logSetupOnce sync.Once
 
+const (
+	srv  = "srv"
+	ver  = "ver"
+	host = "host"
+)
+
 // SetupLogging sets up the default metrics logging.
 func SetupLogging(name, version string) error {
+	return setupLogging(name, version, nil)
```

Comment:
would it make sense to make this an empty map ? Just to be more transparent down the line, instead of the potential need to handle nils. What do you think ?

Reply:
Don't think it makes a difference as the result is the same when iterating an empty map however changed.